shared/menubar/index.desktop.js

```diff
@@ -186,12 +182,14 @@ class MenubarRender extends React.Component<Props, State> {
     )
   }
 
-  _menuView(title: string, iconType: Kb.IconType, count: number) {
+  _menuView(title, iconType, count) {
     return (
-      <Kb.Box2 direction="horizontal" style={{width: '100%'}}>
+      <Kb.Box2 direction="horizontal" fullWidth={true}>
         <Kb.Box style={{marginRight: Styles.globalMargins.xsmall, position: 'relative'}}>
           <Kb.Icon type={iconType} color={Styles.globalColors.black_20} fontSize={20} />
-          {!!count && <Kb.Badge badgeNumber={count} badgeStyle={{left: 14, position: 'absolute', top: -2}} />}
+          {!!count && (
+            <Kb.Badge badgeNumber={count || 0} badgeStyle={{left: 14, position: 'absolute', top: -2}} />
```

Comment:
`count || 0` necessary?

Reply:
dunno i just kept what it had before but moved it here instead of all the callers

Reply:
ah k I asked because L190 asserts `!!count`

Reply:
this thing can allow undefined, i didn't look into badge